File: despatch.Domain/Models/Default/Request/MessageRequest.cs
using System;

namespace despatch.Domain.Models.Default.Request
{
    public class MessageRequest : BaseEntityRequest
    {
        public string Text { get; set; }
        public DateTime SendDate { get; set; }

        public string UserId { get; set; }
        public string RoomId { get; set; }

        public virtual UserRequest User { get; set; }
        public virtual RoomRequest Room { get; set; }
    }
}

File: despatch.WEB/app/src/swagger/models/message-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { RoomRequest } from './room-request';
import { UserRequest } from './user-request';
export interface MessageRequest extends BaseEntityRequest{
  room?: RoomRequest;
  roomId?: string;
  sendDate: string;
  text?: string;
  user?: UserRequest;
  userId?: string;
}
